contracts/governance/extensions/GovernorTimelockAccess.sol

```diff
@@ -223,12 +232,12 @@ abstract contract GovernorTimelockAccess is Governor {
         // If the proposal has been scheduled it will have an ETA and we have to externally cancel
         if (eta != 0) {
             for (uint256 i = 0; i < targets.length; ++i) {
-                (bool delayed, uint32 nonce) = _getManagerData(plan, i);
-                if (delayed) {
-                    // Attempt to cancel considering the operation could have been cancelled and rescheduled already
-                    uint32 canceledNonce = _manager.cancel(address(this), targets[i], calldatas[i]);
-                    if (canceledNonce != nonce) {
-                        revert GovernorMismatchedNonce(proposalId, nonce, canceledNonce);
+                (, bool withDelay, uint32 nonce) = _getManagerData(plan, i);
+                if (withDelay) {
+                    bytes32 operationId = keccak256(abi.encode(address(this), targets[i], calldatas[i]));
```

Comment:
Let's make `hashOperation` a public function in the AccessManager and use it here.